.github/workflows/update-all.yml

```diff
@@ -0,0 +1,68 @@
+## runs all make update-all to update all possible updatable dependencies (ISO/kicbase/Addon Versions)
+name: "update-all"
+on:
+  workflow_dispatch:
+env:
+  GOPROXY: https://proxy.golang.org
+  GO_VERSION: '1.24.0'
+permissions:
+  contents: read
+jobs:
+  update-all:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+      - uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5
+        with:
+          go-version: ${{env.GO_VERSION}}
+      - name: Bump all versions
+        id: bumpVersions
+        run: |
+          set -e
+          deps=$(grep -oP '^\s*"([^"]+)":' hack/update/get_version/get_version.go | sed -E 's/^\s*"//;s/":$//')
+          for dep in $deps; do
+            var=$(echo $dep | tr 'a-z-' 'A-Z_')
+            old=$(DEP=$dep make get-dependency-version | tail -n1)
+            echo "OLD_${var}=$old" >> "$GITHUB_OUTPUT"
+            eval "OLD_${var}=$old"
```

Comment:
What is the purpose of this loop? Same for the next loops.

Reply:
supposed to get the Older version of components before Update
and AFTER update we will check again what is the version,
if before/after versions are different we will print it in the PR description
so ti will be a list of 
bump X version form 1 to 2
bump Y version form 0.2 to 0.3